lib/BootBot.js

```diff
@@ -146,8 +146,8 @@ class BootBot extends EventEmitter {
     .catch(err => console.log(`Error sending message: ${err}`));
   }
 
-  sendThreadRequest(body, method) {
-    return this.sendRequest(body, 'thread_settings', method);
+  sendProfileRequest(body, method) {
+    return this.sendRequest(body, 'messenger_profile', method);
```

Comment:
I'd like to keep the `sendThreadRequest` method around for now (in addition to the new `sendProfileRequest`). I'd add a console log in `sendThreadRequest` warning the user that this method has been deprecated and that it'll be removed in a future release. It's unlikely that anyone is using this method directly, but I don't want to introduce any breaking changes if possible.